video/out/opengl/video.c

```diff
@@ -2464,6 +2464,8 @@ static void pass_colormanage(struct gl_video *p, struct mp_colorspace src, bool
             gl->BindBuffer(GL_SHADER_STORAGE_BUFFER, 0);
         }
 
+        gl_sc_enable_extension(p->sc, "GL_ARB_shader_storage_buffer_object");
+
```

Comment:
Cosmetic: Probably better without the newline. Also should we make gl_sc_ssbo implicitly turn on the extension instead of relying on the user to specify it?